src/oop/special_class_method/medium/q12/QuestionText.md

```diff
@@ -0,0 +1,25 @@
+# oop-special_class_method-medium-q12
+
+### Question Text:
+
+Make a Java class called **Shirt** with the following description:
```

Comment:
Customers love to shop online instead of driving to the store. Department stores need a system to model their current and past inventory so it can be displayed online. Use the requirements below to write a Java class `Shirt` to keep track of clothing in a store.